src/MobileList.js

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect } from 'react';
+import moment from 'moment';
+import groupBy from 'lodash.groupby';
+import sortBy from 'lodash.sortby';
+require('twix');
+
+const MAX_DAYS_IN_LIST = 1;
+
+function EventTimes(props) {
+  const { rawTimes } = props;
+  let sortedTimesByDate = groupBy(sortBy(rawTimes,
+    // Sort all of the ranges by when they start;
+    // Unix returns the millisecond time, so all
+    // events will be different
+    (item) => { return item.start.unix() }),
+    // Group the ranges by the day they happen on;
+    // fully including the year, month, and day
+    // in that order guarantees that normal sorting
+    // will respect 9/31 v 10/1, and 2020/01 vs 2019/12
+    (item) => { return item.start.format('YYYY-MM-DD') }
+  )
+
+  let sortedDates = Object.keys(sortedTimesByDate).sort();
+
+  const dateRowFactory = (date) => {
+    let times = sortedTimesByDate[date];
+    let dayStr = times[0].start.format('ddd M/D')
+    let timeStrs = times.map((time) => time.range.format({ hideDate : true })).join(', ')
+    return (
+      <p key={`time-${date}`}>
+        { dayStr }{' | '}{ timeStrs }
+      </p>
+    )
+  }
+
+  if (sortedDates.length <= MAX_DAYS_IN_LIST) {
+    return sortedDates.map(dateRowFactory)
+  } else {
+    let nextDay = sortedDates[MAX_DAYS_IN_LIST - 1];
+    let lastDay = sortedDates[sortedDates.length - 1];
+    let nextStart = sortedTimesByDate[nextDay][0].start;
+    let lastStart = sortedTimesByDate[lastDay][0].start;
+    return sortedDates.slice(0,MAX_DAYS_IN_LIST - 1).map(dateRowFactory).concat(
+      <p>
+        More Times from {nextStart.twix(lastStart, { allDay : true }).format()}
+      </p>
+    )
+  }
+}
+
+export function MobileList(props){
+
```

Comment:
Could maybe use a quick comment about what this function does. 

I take it this generates jsx for a list of events to be used in the mobile view?